examples/aerial_atomic_swap.py

```diff
@@ -51,6 +51,7 @@ def main():
     bob = LocalWallet(PrivateKey("p0h0sYImB4xGq3Zz+xfIrY4QR6CPqeNg8w6X3NUWLe4="))
```

Comment:
Replace these wallets with newly generated ones now that you are funding them dynamically. 
Apply below as well.